trustfall_core/test_data/tests/frontend_errors/transform_used_on_property.frontend-error.ron

```diff
@@ -0,0 +1 @@
+Err(FoldOnProperty("vowelsInName"))
```

Comment:
Using `@transform` without `@fold` is caught at another place in the code. And because we are not allowing `@fold` on the property, the whole usage of `@transform` is blocked for properties.

Reply:
Would you mind renaming this test case to `fold_and_transform_used_on_property` to avoid confusion?

In principle, `@transform` by itself on properties should be valid, and I'd like to support it in the future. We just currently don't have any implemented property transformations, so `@transform` by itself on properties is therefore also invalid because it's useless.

Here renaming the test case is "leaving room" for `@transform` on properties to have a test case take this name which is more appropriate for it.